src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1530,5 +1531,15 @@ public Location get(LootGenerateEvent event) {
 				}
 			}, EventValues.TIME_NOW);
 		}
+
+		// PlayerDeepSleepEvent
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerDeepSleepEvent")) {
+			EventValues.registerEventValue(PlayerDeepSleepEvent.class, Player.class, new Getter<Player, PlayerDeepSleepEvent>() {
+				@Override
+				public Player get(PlayerDeepSleepEvent event) {
+					return event.getPlayer();
+				}
+			}, EventValues.TIME_NOW);
+		}
```

Comment:
```suggestion
```
Since this event extends `PlayerEvent`, the player event value is automatically registered